styles/npm/catppuccin.user.css

```diff
@@ -1035,9 +1035,14 @@
     }
 
     /* Input elements  */
-    #signup_email_error_message {
+    #signup_email_error_message, #signup_password_error_message {
       color: @red;
     }
+
+    ._6e77ea38 {
```

Comment:
Comment please! Also check here to make sure there isn't a better classname.

Reply:
i already made sure there was no better classname